pyrokinetics/equilibrium.py

```diff
@@ -182,10 +352,13 @@ def get_flux_surface(self, psi_n):
 
         paths = con.collections[1].get_paths()
 
-        if psi_n == 1.0 and len(paths) == 0:
-            raise ValueError(
-                "PsiN=1.0 for LCFS isn't well defined. Try lowering psi_n_lcfs"
-            )
+        if psi_n == 1.0:
+            if len(paths) == 0:
+                raise ValueError(
+                    "PsiN=1.0 for LCFS isn't well defined. Try lowering psi_n_lcfs"
+                )
+            elif len(paths) > 1:
+                paths = [np.concatenate([path for path in paths])]
```

Comment:
This was designed to merge two contours if they didn't form a closed loop. 

This will break the result in #47, need to examine in detail

Reply:
Removed in https://github.com/pyro-kinetics/pyrokinetics/pull/48/commits/f660b0c8adfc49b4ed6a34a999e80b2070a56e00